generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management.component.html

```diff
@@ -30,7 +30,7 @@
     <jhi-alert></jhi-alert>
     <div class="row">
         <%_ if (searchEngine == 'elasticsearch') { _%>
-        <div class="col-sm-12 no-padding-right">
+        <div class="col-sm-12 no-padding-center">
```

Comment:
indeed, it doesn't exist lol :stuck_out_tongue_closed_eyes:
I will simply remove no-padding-right for the same result